### PR TITLE
fix repeated content address example

### PR DESCRIPTION
**Pull request by xiekeyang**
should not be present 2 identicial cas file under a dir.

Signed-off-by: xiekeyang <xiekeyang@huawei.com>

### CODE REVIEW

File: image-layout.md [hunk 1]
```diff
@@ -94,9 +94,10 @@ The blobs directory MAY be missing referenced blobs, in which case the missing b
 ### Example Blobs
 
 ```
-$ cat ./blobs/sha256/e692418e4cbaf90ca69d05a66403747baa33ee08806650b51fab815ad7fc331f | jq
+$ cat ./blobs/sha256/9b97579de92b1c195b85bb42a11011378ee549b02d7fe9c17bf2a6b35d5cb079 | jq
 {
   "schemaVersion": 2,
+  "mediaType": "application/vnd.oci.image.manifest.list.v1+json",
```
Comment (jonboulle) [inline image-layout.md]: No, this was removed in https://github.com/opencontainers/image-spec/pull/411

**Comment by xiekeyang** [comment]
@jonboulle Oh, it is my fault. Updated it, PTAL.

**Comment by vbatts** [comment]
LGTM

**Comment by vbatts** [comment]
... why did the other approval fall away?

**Comment by jonboulle** [comment]
Lgtm

**Comment by jonboulle** [comment]
LGTM

**Pull request merged**